homeassistant/components/google_assistant/trait.py

```diff
@@ -145,6 +150,26 @@ def _google_temp_unit(units):
     return "C"
 
 
+def _next_selected(items: Iterable[str], selected: Optional[str]) -> Optional[str]:
+    """Return the next item in a item list starting at given value.
+
+    If selected is missing in items, None is returned
+    """
+    try:
+        cycle = itertools.cycle(items)
+        current = next(cycle)
+        starting = current
+        while True:
+            if current == selected:
+                break
+            current = next(cycle)
+            if current == starting:
+                return None
+        return next(cycle)
+    except StopIteration:
+        return None
```

Comment:
Seems kinda complicated? What about
```suggestion
    try:
        index = items.index(selected)
    except ValueError:
        return None

    next_item = 0 if index == len(items) - 1 else index + 1
    return items[next_item]
```

Reply:
Yes I suppose so :). Will adjust and merge.